node/cli/src/command.rs

```diff
@@ -1,168 +1,189 @@
-};
-use sc_service::config::{BasePath, PrometheusConfig};
-
-use sp_core::hexdisplay::HexDisplay;
-use sp_runtime::traits::Block as BlockT;
-use std::{io::Write, net::SocketAddr};
-use t3rn_primitives::Block;
-
-#[cfg(feature = "with-parachain-runtime")]
-fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
-    Ok(match id {
-        "dev" => Box::new(chain_spec::development_config()),
-        "circuit" => Box::new(chain_spec::circuit_config()),
-        "" | "local" => Box::new(chain_spec::local_testnet_config()),
-        path => Box::new(chain_spec::ChainSpec::from_json_file(
-            std::path::PathBuf::from(path),
-        )?),
-    })
-}
-
-#[cfg(feature = "with-standalone-runtime")]
-fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
-    Ok(match id {
-        "dev" => Box::new(chain_spec::Alternative::Development.load()),
-        "circuit" => Box::new(chain_spec::Alternative::Development.load()),
-        "" | "local" => Box::new(chain_spec::Alternative::LocalTestnet.load()),
-        path => Box::new(chain_spec::ChainSpec::from_json_file(
-            std::path::PathBuf::from(path),
-        )?),
-    })
-}
-
-impl SubstrateCli for Cli {
-    #[cfg(feature = "with-parachain-runtime")]
-    fn impl_name() -> String {
-        "Circuit Parachain Collator".into()
+pub mod standalone {
+    use crate::standalone::{Cli, Subcommand};
+    use circuit_service::chain_spec::standalone::{self as chain_spec};
+
+    use circuit_service::standalone::{new_full, new_partial, ExecutorDispatch};
+    use circuit_standalone_runtime::VERSION;
+    use sc_cli::{RuntimeVersion, SubstrateCli};
+    use sc_service::PartialComponents;
+    use t3rn_primitives::Block;
+
+    fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+        Ok(match id {
+            "dev" => Box::new(chain_spec::Alternative::Development.load()),
+            "circuit" => Box::new(chain_spec::Alternative::Development.load()),
+            "" | "local" => Box::new(chain_spec::Alternative::LocalTestnet.load()),
+            path => Box::new(chain_spec::ChainSpec::from_json_file(
+                std::path::PathBuf::from(path),
+            )?),
+        })
     }
 
-    #[cfg(feature = "with-standalone-runtime")]
-    fn impl_name() -> String {
-        "Circuit Node".into()
-    }
+    impl SubstrateCli for Cli {
+        fn impl_name() -> String {
+            "Circuit Node".into()
+        }
 
-    fn impl_version() -> String {
-        "0.1.0".into()
-    }
+        fn impl_version() -> String {
+            "0.1.0".into()
+        }
 
-    fn description() -> String {
-        format!(
-            "Circuit Parachain Collator\n\nThe command-line arguments provided first will be \
+        fn description() -> String {
+            format!(
+                "Circuit Parachain Collator\n\nThe command-line arguments provided first will be \
```

Comment:
Is this still correct for the standalone node?

Reply:
Removed incorrect text, thanks